eng/Version.Details.xml

```diff
@@ -9,21 +9,21 @@
 -->
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.AspNetCore.Razor.Language" Version="3.0.0-preview4.19168.1">
+    <Dependency Name="Microsoft.AspNetCore.Razor.Language" Version="3.0.0-preview4.19171.3">
```

Comment:
Do not make version changes with the old version of `darc` or manually. In this case, there are newer coherent versions that will be available tomorrow morning.